api/server/router/swarm/cluster_routes.go

```diff
@@ -178,15 +190,21 @@ func (sr *swarmRouter) getServices(ctx context.Context, w http.ResponseWriter, r
 
 func (sr *swarmRouter) getService(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
 	var insertDefaults bool
+
 	if value := r.URL.Query().Get("insertDefaults"); value != "" {
 		var err error
 		insertDefaults, err = strconv.ParseBool(value)
 		if err != nil {
-			err := fmt.Errorf("invalid value for insertDefaults: %s", value)
```

Comment:
Why remove the error context?

Reply:
unsure. It must've gotten removed by accident.

Reply:
Oh, because directly below, it would get wrapped again, and the result (I think) would be something like `Error: invalid value for insertDefaults: invalid value for insertDefaults: {{ strconv error }}`. Am I incorrect on that?

Reply:
Oh of course :)